### ISSUE TITLE
No section: 'years'？？

### ISSUE THREAD

**Issue by RussiaVk** [Mebus/cupp#34]

**Comment by callmev2016** [comment]
i got this issue too. python 3.7.2

**Comment by axelsimon** [comment]
To be more explicit:
```
$ python3 cupp.py -h
Traceback (most recent call last):
  File "cupp/cupp.py", line 1068, in <module>
    main()
  File "cupp/cupp.py", line 1001, in main
    read_config()
  File "cupp/cupp.py", line 62, in read_config
    "years": config.get("years", "years").split(","),
  File "/usr/lib64/python3.7/configparser.py", line 780, in get
    d = self._unify_values(section, vars)
  File "/usr/lib64/python3.7/configparser.py", line 1146, in _unify_values
    raise NoSectionError(section) from None
configparser.NoSectionError: No section: 'years'
```

**Comment by Mebus** [comment]
This should be fixed now with 0905db74702ed94f5c7190b6d5648643cce7073a !

Thanks.
